packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -154,6 +150,18 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     this._button.option('disabled', state);
   }
 
+  _updateAria(emptyViewId: string | null): void {
+    $(this._textArea.$element())
```

Comment:
I understand that you are avoiding the // @ts-expect-error comment here, but it’s better to use the _input method instead, as we use this method widely. It’s not a big issue if the comment is present; when we move the TextEditor to a native class, the comment will be removed.

https://github.com/marker-dao/DevExtreme/blob/24_2_chat_accessibility/packages/devextreme/js/__internal/ui/text_box/m_text_editor.base.ts#L153

Reply:
Moreover, we can use public option inputAttr here: https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#inputAttr

Reply:
[ddd98c2f8675ba0e9ea72a8e3cde9824787a94a4]